docs/content/docs/guide/statuses.md

```diff
@@ -137,3 +137,25 @@ generating artifacts on every push. The relevant section of the pipeline can be
 found here:
 
 <https://github.com/openshift-pipelines/pipelines-as-code/blob/7b41cc3f769af40a84b7ead41c6f037637e95070/.tekton/push.yaml#L116>
+
+## Tekton Results Annotation
+
+A specialized annotation designed explicitly for use with Tekton Results.
```

Comment:
This seems a bit verbose and chatgpt but that's better than bad english or no doc 🙃

Reply:
(Chatgpt is a great tool, not perfect though🙃) Got it, I'll keep it concise.